lib/Net/SAML2/Protocol/ArtifactResolve.pm

```diff
@@ -1,3 +1,5 @@
+use strict;
```

Comment:
> As we are using Moose, we get `use strict` and `use warnings` for free. I don't think we need to add them. Which is one of the reasons I removed them in [066100f](https://github.com/perl-net-saml2/perl-Net-SAML2/commit/066100f376b75427f45aab7ebc407df8805ad2b8).

I wondered why you removed them I was oging to ask you about that.  I would prefer to keep them since it will cause the various checks everywhere to complain.  I can likely disable the PBP check from dist test but it will show up on cpants, etc.

Unless I am mistaken it won't add much overhead since whether it is likely already loaded by a previous module anyway.

Reply:
I have that covered as well in another PR.